src/groups/mqb/mqbblp/mqbblp_relayqueueengine.cpp

```diff
@@ -1048,6 +1048,14 @@ mqbi::QueueHandle* RelayQueueEngine::getHandle(
         return 0;  // RETURN
     }
 
+    if (!d_queueState_p->canMerge(handleParameters)) {
+        CALLBACK(bmqp_ctrlmsg::StatusCategory::E_REFUSED,
```

Comment:
Strictly speaking, the handle parameters are not INVALID, so it is not correct to return invalid parameters return code. On the other hand, we expose some information about the current configure counters. Should we allow this?
It is possible to write a client that will probe the broker with different counters and depending on the result it will know the exact counters of the handle on the broker side